index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Project -1</title>
+    <link rel="stylesheet" href="style.css">
+    <!-- Use of font Roboto -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+
+    <!-- Use of Font Poppins -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;500&family=Roboto:wght@100&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- For creaing My Logo and Menu -->
+    <header class = "navheader">
+        <div class="logo">My Logo</div>
+        <div class="menubar">
+            <div class ="menu1"></div>
+            <div class ="menu2"></div>
+            <div class ="menu3"></div>
+        </div>
+    </header>
+
+    <!-- For Creating Shapes -->
+
+    <div class="shapespace">
+        <div class="rectangle6"></div>
+        <div class="eclipse11"></div>
+        <div class="eclipse8"></div>
+        <div class="rectangle7"></div>
+        <div class="ellipse5"></div>
+        <div class="ellipse12"></div>
+        <div class="ellipse15"></div>
+        <div class="U"></div>
+        <div class="ellipse16"></div>
+        <img src="images/Ellipse 3.jpg" class = "eclipse3">
+        <div class="rectangle4"></div>
+        <div class="finalrect"></div>
+        <div class="circle"></div>
+
+    </div>
+<!-- Main Heading and Paragraph -->
+    <div class = "main">
+        <h1 class="mainfont"> I'm Mukund <br> Glad to see <br> you!</h1>
+        <p class = "mainpara">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href = "#" target="_self" class="connect">LET'S CONNECT</a>
+    </div>
+<!-- Social Media iocns -->
+    <div class="image">
+     <div class="images"> <img src="images/twitter.jpg" alt="" class="images"> </div>
+     <div class="images">  <img src="images/Linkedin icon.jpg" alt="" class="images"> </div>
+     <div class="images"> <img src="images/M.jpg" alt="" class="images"></div>   
+     <div class="images"> <img src="images/github.jpg" alt="" class="images"> </div>   
+    <div class="images"><img src="images/hands.jpg" alt="" class="images"></div>
```

Comment:
- [ ] It is required to list your social icons with a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Kindly make use of `ul` to group your social icons, list each of them with the use of `li` and each icon should be wrapped in a link `a` tag.
Something like this 
```
    <ul><li><a href="#">Your Icon</a></li></ul>
```